nlp/text_preprocessing.py

```diff
@@ -52,7 +52,7 @@ def preprocess_text(
     numbers_to_words=False,
     remove_numbers=True,
     remove_stopwords=False,
-    language="sk",
+    language=DEFAULT_LANG,
```

Comment:
Why are we changing now the default lang to en from sk?

Reply:
Does it matter if language is passed to the function anyway?

Reply:
I think it now does but since "sk" is not yet supported, it makes sense to change it to English so I think this can be approved.